spec/views/shared/_alert.html.erb_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe 'shared/_alert.html.erb' do
+  it 'renders message from param' do
+    render 'shared/alert', { message: 'FYI' }
+
+    expect(rendered).to have_content('FYI')
+  end
+
+  it 'renders message from block' do
+    render 'shared/alert' do 'FYI' end
```

Comment:
rubocop got mad at this.... gotta use curlies for single-line blocks (or make it a multiline block)

Reply:
> rubocop got mad at this.... gotta use curlies for single-line blocks (or make it a multiline block)

Fixed in 564c69237.